domains/anonymousdeveloper.json

```diff
@@ -0,0 +1,17 @@
+{
+  "owner": {
+    "username": "noah-dev-codes"
+  },
+  "record": {
+    "MX": [
+      "mx1.improvmx.com",
+      "mx2.improvmx.com"
+    ],
+"CNAME": "noah-dev-codes.github.io",
```

Comment:
CI says you cant have both CNAME and MX (hint: you can put MX on a separate subdomain)